### PR TITLE
Added methods for calculation of shear stress and axial stress

### LINKED ISSUE #16366: No function to calculate shear stress and axial stress
In `beam.py` file of `continuum_mechanics` folder there is no function to calculate shear stress and axial stress.

### PR DESCRIPTION
**Pull request by abhinavsagar**
#### References to other Issues or PRs

[FIxes #16366 ](https://github.com/sympy/sympy/issues/16366)

#### Release Notes

* functions
  * added fucntion for calculating shear stress.
  * added fucntion for calculating axial stress.

### CODE REVIEW

**Comment by oscarbenjamin** [comment]
If you look at the travis logs (click details below) you can see that this PR failed the code quality checks with this error:
```
AssertionError: File contains trailing whitespace: /home/travis/build/sympy/sympy/sympy/physics/continuum_mechanics/beam.py, line 1812.
```
You need to fix that before the tests will run on this PR.

It's quicker to run the code quality tests yourself locally with `bin/test quality`.

File: sympy/physics/continuum_mechanics/beam.py [hunk 1]
```diff
@@ -1801,20 +1810,32 @@ def solve_for_reaction_loads(self, *reaction):
 
     def shear_force(self):
         """
-        Returns a list of three expressions which represents the shear force
-        curve of the Beam object along all three axes.
+        Returns a list of three expressions which represents the shear force curve of the Beam object along all three axes.
```
Comment (czgdp1807) [inline sympy/physics/continuum_mechanics/beam.py]: I think the earlier version of this part was good.

File: sympy/physics/continuum_mechanics/beam.py [hunk 2]
```diff
@@ -1801,20 +1810,32 @@ def solve_for_reaction_loads(self, *reaction):
...
         q = self._load_vector
         m = self._moment_load_vector
         return [integrate(-q[0], x), integrate(-q[1], x), integrate(-q[2], x)]
 
+    def shear_stress(self):
+        """
+        Returns a list of three expressions which represents the shear stress
+        curve of the Beam object along all three axes.
+        """  
+        return [self.shear_force()[0]/self.area, self.shear_force()[1]/self.area, self.shear_force()[2]/self.area]
```
Comment (czgdp1807) [inline sympy/physics/continuum_mechanics/beam.py]: Please break this line into two.

Comment (czgdp1807) [reply sympy/physics/continuum_mechanics/beam.py]: > Please break this line into two.

+1

File: sympy/physics/continuum_mechanics/tests/test_beam.py [hunk 3]
```diff
@@ -96,6 +96,11 @@ def test_Beam():
     q = 4000*x/3 - 4*SingularityFunction(x, 0, 3)/3 + SingularityFunction(x, 10, 3) + 60*SingularityFunction(x, 30, 2) + SingularityFunction(x, 30, 3)/3 - 12000
     assert p == q/(E*I)
 
+    # Test for shear stress distribution function
+    p = b1.shear_stress()
+    q = -8*SingularityFunction(x, 0, -2) + 6*SingularityFunction(x, 10, -2) + 120*SingularityFunction(x, 30, -3) + 2*SingularityFunction(x, 30, -2)
```
Comment (czgdp1807) [inline sympy/physics/continuum_mechanics/tests/test_beam.py]: > Please break this line into two.
+1

File: sympy/physics/continuum_mechanics/tests/test_beam.py [hunk 4]
```diff
@@ -601,4 +611,4 @@ def test_parabolic_loads():
     beam.apply_load(P, 0, 8, end=L)
     loading = beam.load.xreplace({L: 10, E: 20, I: 30, P: 40})
     assert loading.xreplace({x: 5}) == 40 * 5**8
-    assert loading.xreplace({x: 15}) == 0
+    assert loading.xreplace({x: 15}) == 0
```
Comment (czgdp1807) [inline sympy/physics/continuum_mechanics/tests/test_beam.py]: Add a new line.

**Comment by czgdp1807** [comment]
@abhinavsagar Are you interested to continue your work in this PR? Can it be closed?

**Comment by czgdp1807** [comment]
@sympy/mechanics Is there any possibility to continue work on this?

**Comment by czgdp1807** [comment]
Closing this as the PR only contains wrapper functions for one liner codes without branching. Thanks for the contributions. Please let us know if you wish to continue and complete this.

**Pull request closed without merging**